lib/org/chromium/webidl/LocalGitRunner.js

```diff
@@ -15,7 +15,7 @@ foam.CLASS({
     'org.chromium.webidl.URLExtractor',
   ],
 
-  imports: [ 'outputDAO?' ],
+  imports: [ 'outputDao? as dao' ],
```

Comment:
Alternatively, if this should be directly injected, we can drop the import, and just use `value: null` on the `outputDAO` property.